NAMESPACE

```diff
@@ -35,7 +35,8 @@ export(install_url)
 export(install_version)
 export(local_package_deps)
 export(package_deps)
-export(parse_github_repo_spec)
+export(parse_github_url)
+export(parse_repo_spec)
```

Comment:
It is an incompatible change to drop `parse_github_repo_spec`, I think we should keep that name (as well?).

Reply:
OK I will address this. Need to re-orient myself to what I was doing!

Just for context, I can't find any evidence of `parse_github_repo_spec()` usage on CRAN, outside of remotes itself.

GitHub-wise, I see one other use:

  * https://github.com/mrc-ide/provisionr/blob/d74f2cd3fc3b59e878439ac8d766b83570b0a681/R/remote.R#L37 by @richfitz

Reply:
Thanks!

Reply:
As requested here: https://github.com/r-lib/remotes/issues/13 😀

Reply:
OK I went back to the old name, @gaborcsardi.

Here's what I think (?) I was thinking:

  * Start to de-emphasize the GitHub-ness, since these repo spec parsers are not too hard to generalize to BitBucket, GitLab, and the like. But it's easy to make this change now and cross that bridge later.